docs: Adding requirements and a component change request for vajson

linnealinnea opened the pull request:
Adapting json architecture doc to include vajson as a new json backend and adding component requirements. Related to #2310.

linnealinnea committed:
Issue 2310 - Component request: (JSON parsing) VaJson in S-CORE

Adapting json architecture doc to include vajson as a new backend and adding component requirements.

linnealinnea committed:
Issue 2310 - Component request: (JSON parsing) VaJson in S-CORE

    Dummy commit to report author details.

    Also-by: Linnea Bennerhed <linnea.bennerhed@vector.com>

linnealinnea committed:
Merge branch `eclipse-score:main` into Issue-2310-jsonparsing-vajson-doc-updates

FScholPer reviewed (changes requested):
Needs agreement in architecture community. Do we have something like that for vajson?

linnealinnea committed:
Issue 2310 - Component request: (JSON parsing) VaJson in S-CORE

Fix Sphinx RST build issues.

Also-by: Linnea Bennerhed <linnea.bennerhed@vector.com>

linnealinnea committed:
Issue 2310 - Component request: (JSON parsing) VaJson in S-CORE

Fix requirement linking.

Also-by: Linnea Bennerhed <linnea.bennerhed@vector.com>

linnealinnea committed:
Update index.rst

Issue 2310 - Component request: (JSON parsing) VaJson in S-CORE

Fix indenting.

Also-by: Linnea Bennerhed <linnea.bennerhed@vector.com>

Signed-off-by: Linnea Bennerhed <72805813+linnealinnea@users.noreply.github.com>

linnealinnea commented:
@FScholPer The component was accepted in the baselib FT [MeetingNotes](https://github.com/orgs/eclipse-score/discussions/1239?sort=new#discussioncomment-15159078), and my understanding is that @4og will bring this up on next architecture community meeting after Christmas.

thilo-schmitt commented:
@linnealinnea thank you for your PR and great to hear that it is already aligned in the Feature Team. Let's, however, get the buy-in from the Architecture Community also. We have to make sure that we uphold our principle "one solution only for any given aspect", which is watched over by the Architecture Community.

linnealinnea committed:
Issue 2310 - Component request: (JSON parsing) VaJson in S-CORE

Adding first version of "Change Management Component Request"

linnealinnea reviewed (commented):
to be removed

linnealinnea committed:
Issue 2310 - Component request: (JSON parsing) VaJson in S-CORE

Fix failing doc build

linnealinnea commented:
Keeping the PR in draft mode until next Architecture Community round

linnealinnea committed:
Issue 2310 - Minor RST style adjustments

Minor style adjustments.

Signed-off-by: Linnea Bennerhed <72805813+linnealinnea@users.noreply.github.com>

linnealinnea commented on docs/modules/baselibs/json/docs/vajson/index.rst:
```diff
@@ -0,0 +1,173 @@
+   :safety: ASIL_B
+   :security: NO
+   :realizes: wp__cmpt_request
+   :tags: contribution_request, feature_request
```
Fixed in latest commit (409de4bd1c27268408811279aeb499e49225f97b)

4og commented:
@JuriSchroeder, can you please take over this PR?

JuriSchroeder commented:
> @JuriSchroeder, can you please take over this PR?

As discussed I'll take over this PR and move it to baselibs.
